data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1217 @@
+				
+	on complete
+		event "Etarip Delivery 5" 8 12
+		conversation
+			`Alexandra's coordinates lead you to a remote landing strip just outside of <planet>'s largest metropolitan area. You land a short distance away from a small transport shuttle and greet the well-dressed group wating in the field. Most appear to be guards or professional workers. One larger, balding man greets you cordially, smiling and clapping you on the shoulder.`
+			`	"Greetings, greetings! I hear you've got some merchandise for us?"`
+			choice
+				`	"Yes, sir. Laser rifles, stun guns... Boom-boom sticks through and through."`
+				`	"Yeah, everything I was told to pick up... I'm sorry, but why are we meeting out here instead of the spaceport?"`
+					goto why
+			`	The man chuckles. "I like it - professional, but casual. Thanks for making your way out here." Two of his men break off and begin transferring cargo out of your ship.`
+				goto meetup1
+					
+			label why
+			`	The man gestures to his men to begin transferring cargo, then crosses his arms and looks off toward the outskirts of the city. "Friction, basically. We used to bring everything in through the spaceport, but we had a few misfires, and the authorities can't keep up with the demand in these urban airports. It takes *ages* to process everything. Plus, you never know what kind of city thugs might be hanging around, waiting to cause trouble."`
+			label meetup1
+			`	A female guard approaches a few moments later. "We've got everything, sir. It looks clean." The man smiles and shakes your hand. "Pleasure doing business, captain - we won't keep you any longer." The small group boards the transport as you head back to your ship.`
+			
+event "Defer Etarip Delivery 4"
+#* * *  (Mission 4 END)  * * *
+
+
+
+#* * * * *  (Mission 5)  * * * * *
+event "Etarip Delivery 5"
+
+mission "Etarip Delivery [5]"
+	name "Delivery to <planet>"
+	description "Deliver <cargo> to <destination> for Etarip Enterprises."
+	blocked `Alexandra contacts you just as you touch down on <origin>. "We have a job for you, <first>, but you're going to need <capacity>. Remember - we have a contract! Drop whatever you're doing and come back."`
+	landing
+	cargo "Baking Supplies" 12
+	deadline 2
+	
+	stopover
+		attributes "farming"
+		distance 1 4
+	
+	to offer
+		has "Etarip Delivery [4]: done"
+		has "event: Etarip Delivery 5"
+		
+	source
+		government Republic
+	destination
+		attributes "paradise"
+		attributes "rich"
+	
+	on offer
+		conversation
+			`You've developed a bit of a phobia for Alexendra calling you unexpectedly just as you arrive on planets. Sure enough, a transmission arrives, and her crisp voice comes over the comm line. "Hey there, <first>! You ready for your last cargo run?"`
+			choice
+				`	"What would I be in for?"`
+					goto business
+				`	"If it let me hear more of your voice, I'd eat Zeppadrogon Caviar."`
+			`	Alexandra laughs. "Disgusting, but charming! It's funny you mention that, though..."`

```

Comment:
> but charming!

Isn't she pretty sick of your groanworthy pick-up lines at this point?

Reply:
Oh, certainly. She was probably sick of them to begin with. But it's also her job to respond somewhat positively to even the worst of pick-up lines; she is attempting to bamboozle the player into smuggling, after all. The best way to keep a flirtatious captain on the hook is to make them feel like they're making progress!